adit/core/utils/dicom_to_nifti_converter.py

```diff
@@ -0,0 +1,45 @@
+import subprocess
+from pathlib import Path
+from typing import Union
+
+
+class DicomToNiftiConverter:
+    def __init__(self, dcm2niix_path: str = "dcm2niix"):
+        """
+        Initialize the converter with the path to the dcm2niix executable.
+        :param dcm2niix_path: Path to the dcm2niix executable.
```

Comment:
I know ... there are not a lot of docstrings in the code 😞, but those that are there are in Google style: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html